<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="activity_tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="overview_tab">
      <attribute name="title">
       <string>Overview</string>
      </attribute>
      <layout class="QGridLayout" name="overview_tab_layout">
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="date_activity_type_area">
         <item>
          <widget class="QLabel" name="date_time_label">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>[Date]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="activity_type_label">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>[Type]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="InfoTable" name="info_table">
         <property name="minimumSize">
          <size>
           <width>310</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>310</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="activity_text_area">
         <item>
          <layout class="QHBoxLayout" name="activity_name_and_flags">
           <item>
            <widget class="QLabel" name="activity_name_label">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>[Activity Name]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="flags_label">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>[Flags]</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="description_label">
           <property name="text">
            <string>[Description]</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" rowspan="2">
        <layout class="QGridLayout" name="map_container"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="data_tab">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="data_tab_layout">
       <item>
        <layout class="QVBoxLayout" name="graphs_layout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="splits_tab">
      <attribute name="title">
       <string>Splits</string>
      </attribute>
      <layout class="QVBoxLayout" name="splits_tab_layout">
       <item>
        <widget class="SplitTable" name="split_table">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="zones_tab">
      <attribute name="title">
       <string>Zones</string>
      </attribute>
      <layout class="QVBoxLayout" name="zones_tab_layout">
       <item>
        <widget class="QtCharts::QChartView" name="zones_graph"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="curve_tab">
      <attribute name="title">
       <string>Curve</string>
      </attribute>
      <layout class="QHBoxLayout" name="curve_tab_layout">
       <item>
        <widget class="QtCharts::QChartView" name="curve_graph"/>
       </item>
       <item>
        <widget class="CurveTable" name="curve_table">
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView&gt;
#include &lt;QtCharts/chartsnamespace.h</header>
  </customwidget>
  <customwidget>
   <class>InfoTable</class>
   <extends>QTableWidget</extends>
   <header>activate/app/tables.h</header>
  </customwidget>
  <customwidget>
   <class>SplitTable</class>
   <extends>QTableWidget</extends>
   <header>activate/app/tables.h</header>
  </customwidget>
  <customwidget>
   <class>CurveTable</class>
   <extends>QTableWidget</extends>
   <header>activate/app/tables.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>activity_tabs</sender>
   <signal>currentChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>update_page()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_page()</slot>
 </slots>
</ui>
